File: source/examples/Relationships/ToOne/ToOne.swift
class Person: Object {
    dynamic var name: String = ""
    dynamic var birthdate: Date = Date(timeIntervalSince1970: 1)
    dynamic var Dog: Dog? = nil
}

class Dog: Object {
    dynamic var name: String = ""
    dynamic var age: Int = 0
    dynamic var breed: String? = nil
}

File: source/examples/Relationships/ToOne/ToOne.kt
class Person: RealmObject {
    var name: String = ""
    var birthdate: Date = Date(milliseconds: 1)
    var dog: Dog? = null
}

class Dog: RealmObject {
    var name: String = ""
    var age: int = 0
    var breed: String? = null
}
